Practical Example/RW_interview_prep2/AttributeArticle+CoreDataProperties.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AttributeArticle+CoreDataProperties.swift
+//  RW_interview_prep
+//
+//  Created by Duc Dang on 8/31/20.
+//  Copyright © 2020 Duc Dang. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+
+extension AttributeArticle {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<AttributeArticle> {
+        return NSFetchRequest<AttributeArticle>(entityName: "AttributeArticle")
+    }
+
+    @NSManaged public var card_artwork_url: String?
+    @NSManaged public var content_type: String?
+    @NSManaged public var contributor_string: String?
+    @NSManaged public var description_plain_text: String?
+    @NSManaged public var descriptionn: String?
```

Comment:
There's a typo here ("descriptionn" with to ns). This is a tiny nitpick, but an employer might use this against you. :]

Reply:
Because it conflicts with the -description method in NSObject so I can not name it as "description". Therefore, my simple brain just added another "n" for this particular situation.